Readme.md
# Hello There Fellow Repository Explorer
## Welcome to my little repository of CTF exercise
Here I post my solves and how i solved the CTF problems on picoCTF.
Explore,Enjoy(Star it if you find it helpfull)




MilkSlap/history.txt
    1  hasdeep -v
    2  hashdeep -v
    3  hasdeep -v
    4  hashdeep -v
    5  ls
    6  ls -l
    7  pwd
    8  whoami
    9  nc -v
   10  nc -v 10.19.42.91
   11  man nc
   12  clear
   13  nano script.sh
   14  ls
   15  chmod +x script.sh
   16  ./script.sh
   17  nano script.sh
   18  chmod +x script.sh
   19  nano script.sh
   20  ./script.sh
   21  clear
   22  wget https://mercury.picoctf.net/static/5eb456e480e485183c9c1b16952c6eda/dolls.jpg
   23  ls
   24  dolls.pg
   25  doll
   26  dolls.jpg
   27  nano dolls.pg
   28  nano dolls.jpg
   29  ren dolls.jpg output.rar
   30  unzip dolls.jpg
   31  ls
   32  cd base_images
   33  ls
   34  unzip 2_c.jpg
   35  ls
   36  cd base_images
   37  ls
   38  unzip 3_c.jpg
   39  ls
   40  cd base_images
   41  ls
   42  unzip 4_c.jpg
   43  ls
   44  cat flag.txt'
   45  nano flag.txt
   46  bvi flag.txt
   47  ls
   48  bvi flag.txt
   49  nano new.txt
   50  clear
   51  wget https://mercury.picoctf.net/static/d0129ad98ba9258ab59e7700a1b18c14/tunn3l_v1s10n
   52  cd
   53  wget https://mercury.picoctf.net/static/d0129ad98ba9258ab59e7700a1b18c14/tunn3l_v1s10n
   54  clear
   55  nano tunn3l_v1s10n
   56  cat tunn3l_v1s10n
   57  sudo apt install ./code_1.67.1-1651841865_amd64.deb
   58  exiftool
   59  clear
   60  exiftool tunn3l_v1s10n
   61  chmod +x tunn3l_v1s10n
   62  ./tunn3l_v1s10n
   63  which python
   64  which python3
   65  python3
   66  clear
   67  vscode
   68  ls
   69  cd Downloads
   70  ls
   71  cd
   72  cd Desktop
   73  ls
   74  cd Do
   75  cd Downloads
   76  ls
   77  clear
   78  mkdir PythonCodes
   79  clear
   80  python3 exponentials.py
   81  python3 modulo.py
   82  clear
   83  python3 string.py
   84  python3 strings.py
   85  vscode
   86  code
   87  hex eitor
   88  hex editor
   89  clear
   90  ls
   91  bvi tunn3l_v1s10n
   92  clear
   93  covert tunn3l_v1s10n -o tunn3l_v1s10n.bmp
   94  convert tunn3l_v1s10n -o tunn3l_v1s10n.bmp
   95  ls
   96  bvi tunn3l_v1s10n
   97  wget https://jupiter.challenges.picoctf.org/static/43c4743b3946f427e883f6b286f47467/garden.jpg
   98  ls
   99  view garden.jpg
  100  wget https://jupiter.challenges.picoctf.org/static/43c4743b3946f427e883f6b286f47467/garden.jpg
  101  ls
  102  view garden.jpg
  103  ls
  104  nano garden.jpg
  105  bvi garden.jpg
  106  python3
  107  clear
  108  python3
  109  clea
  110  clear
  111  ls
  112  Scripts
  113  mkdir Scripts
  114  clear
  115  cd Scripts
  116  clear
  117  nano B2A.py
  118  python3 B2A.py
  119  clear
  120  nano B2A.py
  121  python3 B2A.py\nclear
  122  python3 B2A.py
  123  clear
  124  nano B2A.py
  125  python3 B2A.py
  126  clear
  127  nano O2A.py
  128  python3 O2A.py
  129  nano O2A.py
  130  python3 O2A.py
  131  clear
  132  nano H2A/py
  133  nano H2A.py
  134  python3 H2A.py
  135  clear
  136  python3 H2A.py
  137  cd Scripts
  138  clear
  139  python3 H2A.py
  140  cd Scripts
  141  clear
  142  python3 O2A.py
  143  ls
  144  cd pico
  145  ls
  146  cd ..
  147  cd Scripts
  148  clear
  149  python3 B2A.py
  150  sudo apt install git
  151  sudo apt-get dist-upgrade -y
  152  clera
  153  clear
  154  code
  155  ls
  156  git config --global user.name "AshRahman"
  157  git config --global user.email "ashfaqurrahmanrahat@outlook.com"
  158  clear
  159  wget https://artifacts.picoctf.net/c/79/level5.py
  160  wget V
  161  wget https://artifacts.picoctf.net/c/79/level5.flag.txt.enc
  162  wget https://artifacts.picoctf.net/c/79/level5.hash.bin
  163  wget https://artifacts.picoctf.net/c/79/dictionary.txt
  164  clear
  165  ls
  166  python3 level5.py
  167  clear
  168  cat dictionary.txt
  169  nano level5.py
  170  clear
  171  nano trim.py
  172  python3 level5.py
  173  clear
  174  pyhton3 trim.py
  175  python3 trim.py
  176  nano trim.py
  177  clear
  178  nano trim.py
  179  python3 trim.py
  180  cat dictionary.txt
  181  clear
  182  nano trim.py
  183  python3 trim.py
  184  clear
  185  nano trim.py\nclear
  186  nano trim.py\nclear
  187  nano trim.py
  188  python3 trim.py
  189  clear
  190  nano trim.py
  191  clear
  192  python3 trim.py\npython3 trim.py
  193  nano trim.py
  194  python3 trim.py
  195  nano trim.py
  196  python3 trim.py
  197  celar
  198  clear
  199  nano trim.py
  200  python3 trim.py
  201  clear
  202  nano trim.py
  203  python3 trim.py
  204  clear
  205  nano trim.py
  206  python3 trim.py
  207  clear
  208  nano trim.py
  209  python3 trim.py
  210  nano trim.py
  211  ls
  212  cd picoCTF
  213  ls
  214  cd PW\ Crack\ 5
  215  ls
  216  nano level5.py
  217  clear
  218  nano level5.py
  219  python3 level5.py
  220  nano level5.py
  221  python3 level5.py
  222  nano level5.py
  223  python3 level5.py
  224  nano level5.py
  225  python3 level5.py
  226  nano level5.py
  227  python3 level5.py
  228  nano level5.py
  229  python3 level5.py
  230  clear
  231  nano level5.py
  232  python3 level5.py
  233  nano level5.py
  234  cd picoCTF/PW\ Crack\ 5
  235  ls
  236  wget https://artifacts.picoctf.net/c/79/level5.py
  237  wget V
  238  wget https://artifacts.picoctf.net/c/79/level5.flag.txt.enc
  239  wget https://artifacts.picoctf.net/c/79/level5.hash.bin
  240  wget https://artifacts.picoctf.net/c/79/dictionary.txt
  241  clear
  242  ls
  243  python3 level5.py
  244  clear
  245  cat dictionary.txt
  246  nano level5.py
  247  clear
  248  nano trim.py
  249  python3 level5.py
  250  clear
  251  pyhton3 trim.py
  252  python3 trim.py
  253  nano trim.py
  254  clear
  255  nano trim.py
  256  python3 trim.py
  257  cat dictionary.txt
  258  clear
  259  nano trim.py
  260  python3 trim.py
  261  clear
  262  nano trim.py\nclear
  263  nano trim.py\nclear
  264  nano trim.py
  265  python3 trim.py
  266  clear
  267  nano trim.py
  268  clear
  269  python3 trim.py\npython3 trim.py
  270  nano trim.py
  271  python3 trim.py
  272  nano trim.py
  273  python3 trim.py
  274  celar
  275  clear
  276  nano trim.py

FileType/Flag.sh
#!/bin/sh
# This is a shell archive (produced by GNU sharutils 4.15.2).
# To extract the files from this archive, save it to some FILE, remove
# everything before the '#!/bin/sh' line above, then type 'sh FILE'.

#
lock_dir=_sh00046
# Made on 2022-03-15 06:50 UTC by <root@2751d08abaab>.
# Source directory was '/app'.
#
# Existing files will *not* be overwritten, unless '-c' is specified.
#
# This shar contains:
# length mode       name
# ------ ---------- ------------------------------------------
#   1092 -rw-r--r-- flag
#
MD5SUM=${MD5SUM-md5sum}
f=`${MD5SUM} --version | egrep '^md5sum .*(core|text)utils'`
test -n "${f}" && md5check=true || md5check=false
${md5check} || \
  echo 'Note: not verifying md5sums.  Consider installing GNU coreutils.'
if test "X$1" = "X-c"
then keep_file=''
else keep_file=true
fi
echo=echo
save_IFS="${IFS}"
IFS="${IFS}:"
gettext_dir=
locale_dir=
set_echo=false

for dir in $PATH
do
  if test -f $dir/gettext \
     && ($dir/gettext --version >/dev/null 2>&1)
  then
    case `$dir/gettext --version 2>&1 | sed 1q` in
      *GNU*) gettext_dir=$dir
      set_echo=true
      break ;;
    esac
  fi
done

if ${set_echo}
then
  set_echo=false
  for dir in $PATH
  do
    if test -f $dir/shar \
       && ($dir/shar --print-text-domain-dir >/dev/null 2>&1)
    then
      locale_dir=`$dir/shar --print-text-domain-dir`
      set_echo=true
      break
    fi
  done

  if ${set_echo}
  then
    TEXTDOMAINDIR=$locale_dir
    export TEXTDOMAINDIR
    TEXTDOMAIN=sharutils
    export TEXTDOMAIN
    echo="$gettext_dir/gettext -s"
  fi
fi
IFS="$save_IFS"
if (echo "testing\c"; echo 1,2,3) | grep c >/dev/null
then if (echo -n test; echo 1,2,3) | grep n >/dev/null
     then shar_n= shar_c='
'
     else shar_n=-n shar_c= ; fi
else shar_n= shar_c='\c' ; fi
f=shar-touch.$$
st1=200112312359.59
st2=123123592001.59
st2tr=123123592001.5 # old SysV 14-char limit
st3=1231235901

if   touch -am -t ${st1} ${f} >/dev/null 2>&1 && \
     test ! -f ${st1} && test -f ${f}; then
  shar_touch='touch -am -t $1$2$3$4$5$6.$7 "$8"'

elif touch -am ${st2} ${f} >/dev/null 2>&1 && \
     test ! -f ${st2} && test ! -f ${st2tr} && test -f ${f}; then
  shar_touch='touch -am $3$4$5$6$1$2.$7 "$8"'

elif touch -am ${st3} ${f} >/dev/null 2>&1 && \
     test ! -f ${st3} && test -f ${f}; then
  shar_touch='touch -am $3$4$5$6$2 "$8"'

else
  shar_touch=:
  echo
  ${echo} 'WARNING: not restoring timestamps.  Consider getting and
installing GNU '\''touch'\'', distributed in GNU coreutils...'
  echo
fi
rm -f ${st1} ${st2} ${st2tr} ${st3} ${f}
#
if test ! -d ${lock_dir} ; then :
else ${echo} "lock directory ${lock_dir} exists"
     exit 1
fi
if mkdir ${lock_dir}
then ${echo} "x - created lock directory ${lock_dir}."
else ${echo} "x - failed to create lock directory ${lock_dir}."
     exit 1
fi
# ============= flag ==============
if test -n "${keep_file}" && test -f 'flag'
then
${echo} "x - SKIPPING flag (file already exists)"

else
${echo} "x - extracting flag (text)"
  sed 's/^X//' << 'SHAR_EOF' | uudecode &&
begin 600 flag
M(3QA<F-H/@IF;&%G+R`@("`@("`@("`@,"`@("`@("`@("`@,"`@("`@,"`@
M("`@-C0T("`@("`Q,#(T("`@("`@8`K'<6D`&+RD@0`````!````,&(_-P4`
M``#\`69L86<``$)::#DQ05DF4UG'(\VD```=?___9]]9F_E#_OO_V?OYM_)_
M^]E_-*N=Z_XW__?_F_M=[[`!&QB.@`&@`#0!H`-`R``T`>H:``T-```T```&
M0-`&U!DT,FFTC3U'J>33U3U$`!M0T-`/4`````&@!HS4TT&@--`&@!H-J`VD
M`\H>@C3:F@9/4```:-!Z@JI^J`#(::`T:-!HT`T`::`T`:!D#"`T&@#)B!D#
M0&(:!@@!IB&1H`&@`@"``:QW<`%0^<#-`H'%O"!X""9>>CLAD.(^[PL6/A-B
ME6;]I0QU'(^*<M3A4G020JTX$V:S)<UBPJJ^GA05BX1&=/GP?0Q[D'+!CD+P
MV_K+I4K!/7/[]-KD0PR"2X$3<7S15]YM6_(0H3E!5-0!0@?S%M7+4@%KW"`W
M9TXF\AOG*PC:URG`MOO+*RJ="HSSJ-6/DB)$)!9J4C@F+*0#WWI-'H!8/V/.
MN"&.IB25D82>88[8^:1U<C>D_#Z*'38P5K.XK"R0U$BFCIM2#VV_(5N+2"$,
M^0E%Y%8Z1>T,:6*-JLZ>'\T^3?74((H3*_@S"H1E6[0DKAA*00(S#W`"JC<*
M6:^6M[N<>27G/R`1BRH($\$2[("$7&2->AHJ\X`#>0P,$L`J5,#,*/%1BD1!
MO!RC4\T6>?$ZM`$K_%W)%.%"0QR/-I#'<0`````````````!``````````L`
M`````%1204E,15(A(2$`````````````````````````````````````````
M````````````````````````````````````````````````````````````
M````````````````````````````````````````````````````````````
M````````````````````````````````````````````````````````````
M````````````````````````````````````````````````````````````
M````````````````````````````````````````````````````````````
M````````````````````````````````````````````````````````````
M````````````````````````````````````````````````````````````
M````````````````````````````````````````````````````````````
M````````````````````````````````````````````````````````````
,````````````````
`
end
SHAR_EOF
  (set 20 22 03 15 06 50 39 'flag'
   eval "${shar_touch}") && \
  chmod 0644 'flag'
if test $? -ne 0
then ${echo} "restore of flag failed"
fi
  if ${md5check}
  then (
       ${MD5SUM} -c >/dev/null 2>&1 || ${echo} 'flag': 'MD5 check failed'
       ) << \SHAR_EOF
2bea16cd5c05e4d924fa0d41891a6214  flag
SHAR_EOF

else
test `LC_ALL=C wc -c < 'flag'` -ne 1092 && \
  ${echo} "restoration warning:  size of 'flag' is not 1092"
  fi
fi
if rm -fr ${lock_dir}
then ${echo} "x - removed lock directory ${lock_dir}."
else ${echo} "x - failed to remove lock directory ${lock_dir}."
     exit 1
fi
exit 0
